app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    my_cash = {}
+
+    def inner(*args) -> str:
+        if args not in my_cash.keys():
```

Comment:
Using `args not in my_cash` is sufficient and more Pythonic than `args not in my_cash.keys()`. The `keys()` method is unnecessary here.